Misc/NEWS.d/next/Library/2024-07-22-23-18-57.gh-issue-121237.WkbIpy.rst

```diff
@@ -0,0 +1 @@
+Accept "%:z" in strptime
```

Comment:
When releasing, changes will be added to the changelog. You should describe your changes in detail but keep it concise. For example: 
```
Added %:Z used for XXX to strptime.
```
(I'm sorry, I don't know what %:z does).